example/main.tf

```diff
@@ -139,7 +139,7 @@ terraform {
 }
 
 module demo_cf {
-  source                         = "git::https://github.com/jmgreg31/terraform_aws_cloudfront.git?ref=v4.1.1"
+  source                         = "git::https://github.com/schammah/terraform-aws-cloudfront.git?ref=chammas/enhancements"
```

Comment:
with new pattern of CI, please update this to `git::https://github.com/jmgreg31/terraform_aws_cloudfront.git?ref=staging`

Reply:
done